forge-gui/res/cardsfolder/upcoming/marchesa_dealer_of_death.txt

```diff
@@ -0,0 +1,8 @@
+Name:Marchesa, Dealer of Death
+ManaCost:U B R
+Types:Legendary Creature Human Rogue
+PT:3/4
+T:Mode$ CommitCrime | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDig | TriggerDescription$ Whenever you commit a crime, you may pay {1}. If you do, look at the top two cards of your library. Put one of them into your hand and the other into your graveyard. (Targeting opponents, anything they control, and/or cards in their graveyards is a crime.)
+SVar:TrigDig:AB$ Dig | Cost$ 1 | DigNum$ 2 | ChangeNum$ 1 | DestinationZone2$ Graveyard | NoReveal$ True
+DeckHas:Ability$Graveyard
```

Comment:
I think DeckHints here, because this card fills the graveyard but doesn't actually do anything with them? I could be wrong though 😅

Reply:
Had a look. There is a number of other cards with a similar effect like [Akal Pakal, First Among Equals](https://scryfall.com/card/lci/44/akal-pakal-first-among-equals) or [Nagging Thoughts](Nagging Thoughts) that do have the `DeckHas:Ability$Graveyard` tag. It's fair that probably just as many don't.

Reply:
Good enough for now then! @Agetian may want to weigh in on this topic